tests/e2e/tests/product-gallery/inner-blocks/product-gallery-thumbnails/product-gallery-thumbnails.block_theme.side_effects.spec.ts

```diff
@@ -60,22 +62,38 @@ test.describe( `${ blockData.name }`, () => {
 		const thumbnailsBlock = await pageObject.getThumbnailsBlock( {
 			page: 'editor',
 		} );
-		const largeImageBlock = await pageObject.getMainImageBlock( {
-			page: 'editor',
-		} );
-
-		const thumbnailsBlockBoundingRect = await thumbnailsBlock.boundingBox();
-		const largeImageBlockBoundingRect = await largeImageBlock.boundingBox();
 
 		await expect( thumbnailsBlock ).toBeVisible();
-		// Check the default position: on the left of the large image
-		await expect( thumbnailsBlockBoundingRect?.y ).toBeGreaterThan(
-			largeImageBlockBoundingRect?.y as number
+
+		// We should refactor this.
+		// eslint-disable-next-line playwright/no-wait-for-timeout
+		await page.waitForTimeout( 500 );
```

Comment:
If you will not fix it this PR, please create a follow-up issue!

Reply:
Created! https://github.com/woocommerce/woocommerce-blocks/issues/11939